server/lib/src/idm/ldap.rs

```diff
@@ -643,15 +642,51 @@ mod tests {
         let pce = UnixPasswordChangeEvent::new_internal(UUID_ADMIN, TEST_PASSWORD);
 
         assert!(idms_prox_write.set_unix_account_password(&pce).is_ok());
-        assert!(idms_prox_write.commit().is_ok());
+        assert!(idms_prox_write.commit().is_ok()); // Committing all configs
 
-        let anon_t = ldaps.do_bind(idms, "", "").await.unwrap().unwrap();
+        // Test for optional use of POSIX password for bind
+        // Empty UNIX_PW bind, should allow :
```

Comment:
Comment here doesn't seem to match the test? Should this mean "allow_unix_pw_bind unset - > binds allowed"?

Reply:
Yes, teh variables names are wrong, making them right